### ISSUE TITLE
Documentation update to include short description of data sets

### ISSUE THREAD

**Issue by ljhwang** [geodynamics/Rayleigh#448]
The documentation includes links to available data -Accessing Available Datasets.  This could be improved by adding a short description of what these datasets are.

See: https://rayleigh-documentation.readthedocs.io/en/latest/doc/source/accessing_and_sharing_data.html

**Comment by feathern** [comment]
Indeed.  This got lost in the mix when we were trying to overcome the build issues with the documentation that set in near the end of the workshop.   I'll try to add some descriptive text soon.
